pkg/armhelpers/compute.go

```diff
@@ -2,24 +2,116 @@ package armhelpers
 
 import (
 	"github.com/Azure/azure-sdk-for-go/arm/compute"
+	"github.com/Azure/go-autorest/autorest"
 )
 
 // ListVirtualMachines returns (the first page of) the machines in the specified resource group.
-func (az *AzureClient) ListVirtualMachines(resourceGroup string) (compute.VirtualMachineListResult, error) {
-	return az.virtualMachinesClient.List(resourceGroup)
+func (az *AzureClient) ListVirtualMachines(resourceGroup string) (result compute.VirtualMachineListResult, err error) {
+	client := az.virtualMachinesClient
```

Comment:
Put a comment on each of these calls, to say replace with the one liner, once they added the accept language header.